packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -1087,6 +1087,8 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             { id: 'list.hoverBackground', defaults: { dark: '#2A2D2E', light: '#F0F0F0' }, description: 'List/Tree background when hovering over items using the mouse.' },
             { id: 'list.hoverForeground', description: 'List/Tree foreground when hovering over items using the mouse.' },
             { id: 'list.filterMatchBackground', defaults: { dark: 'editor.findMatchHighlightBackground', light: 'editor.findMatchHighlightBackground' }, description: 'Background color of the filtered match.' },
+            { id: 'tree.indentGuidesStroke', defaults: { dark: '#585858', light: '#a9a9a9', hc: '#a9a9a9' }, description: "Tree Widget's stroke color for active indent guides." },
```

Comment:
Don't they already contributed by Monaco? We should not copy Monaco stuff.

Reply:
@akosyakov I was not able to find the colors in [monaco](https://github.com/microsoft/monaco-editor/blob/daabff0021fd6d4f5b57479b99b9a8ab900f951a/test/playground.generated/customizing-the-appearence-exposed-colors.html). The colors are from [vscode](https://github.com/microsoft/vscode/blob/9198116dede357d2963835cd98f72303a94b081f/src/vs/platform/theme/common/colorRegistry.ts#L373).

Reply:
It is provided by Monaco, please remove:
<img width="1774" alt="Screenshot 2020-07-29 at 09 48 29" src="https://user-images.githubusercontent.com/3082655/88772305-e1a61b00-d180-11ea-93b7-efc50980ad6c.png">

Where do you get indentGuidesStrokeHover? I don't see VS Code having or making use of it at all.

Reply:
@akosyakov It is present in vscode. I added this color because I couldn't find the exact logic that vscode used to render the indent stroke on hover (but I did find the exact color they used). So I added `indentGuidesStrokeHover` and it uses that color.